instrumentation/database/sql/splunksql/config.go

```diff
@@ -121,6 +123,12 @@ func WithTracerProvider(tp trace.TracerProvider) Option {
 	return optionConv{iOpt: internal.WithTracerProvider(tp)}
 }
 
+// WithMeterProvider returns an Option that sets the MeterProvider used with
+// this instrumentation library.
+func WithMeterProvider(mp metric.MeterProvider) Option {
```

Comment:
I'm hesitant to add this to the exported types given this package is stably release.

Reply:
I agree. I will address it.

Reply:
The other possibility is to wait with merging until OTel metrics API is stable.

Reply:
Agreed to unexport it.

Reply:
I will not have access in `test` module to unexported fields...

I think we can just wait until GA of the metrics API. @MrAlias, what do you think?

Reply:
Addressed by:

- e9aebc48
- https://github.com/signalfx/splunk-otel-go/issues/1976